packages/core/src/components/Tabs/Tabs.jsx

```diff
@@ -74,7 +94,24 @@ export class Tabs extends React.PureComponent {
     this.replaceState(href);
   }
 
-  // Filter children and return only TabPanel components
+  handleTabKeyPress(evt, panelId) {
+    switch (evt.key) {
+      case LEFT_ARROW:
+        this.tabSwitchLeft(evt, panelId);
```

Comment:
I believe we'd also want to call `evt.preventDefault()` here (and in the next `case`), otherwise the left and right arrow will still scroll the page horizontally if the page overflows the x-axis. Resize the page to `775px` to see what I mean.